easybuild/easyconfigs/f/festival/festival-2.5.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,129 @@
+easyblock = 'ConfigureMake'
+
+name = 'festival'
+version = '2.5.0'
+
+homepage = ['http://festvox.org/festival/']
+
+description = """
+University of Edinburgh's Festival Speech Synthesis Systems is a free software
+multi-lingual speech synthesis workbench that runs on multiple-platforms
+offering black box text to speech, as well as an open architecture for
+research in speech synthesis.
+It designed as a component of large speech technology systems.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [
+    'http://festvox.org/packed/festival/%(version_major_minor)s',
+    'http://festvox.org/packed/festival/%(version_major_minor)s/voices',
+]
+sources = []
+local_sources = [
+    '%(name)s-%(version)s-release.tar.gz',
+    'festlex_CMU.tar.gz',
+    'festlex_OALD.tar.gz',
+    'festlex_POSLEX.tar.gz',
```

Comment:
Oh my, none of these are versioned at all? :(

Should we rename them on download and include the (major/minor) version to avoid headaches when we update this later?

```
{'download_filename': 'festlex_POSLEX.tar.gz', 'filename': 'festlex_POSLEX-%(version_major_minor)s.tar.gz'},